### PR TITLE
Feature: added azb spark init command

### LINKED ISSUE #74: azure spark init

### PR DESCRIPTION
**Pull request by jafreck**
Fix #74

### CODE REVIEW

File: docs/00-getting-started.md [hunk 1]
```diff
@@ -53,15 +59,17 @@ Once you have one, simply log in and go to the [Azure Portal](https://portal.azu
 
 4. Save your account credentials into the secrets.yaml file
 
-- Copy the secrets.yaml.template file to secrests.cfg
+- Copy the secrets.yaml.template file in the .thunderbolt/ directory to .thunderbolt/secrets.yaml
```
Comment (timotheeguerin) [inline docs/00-getting-started.md]: remove this?

File: docs/00-getting-started.md [hunk 2]
```diff
@@ -53,15 +59,17 @@ Once you have one, simply log in and go to the [Azure Portal](https://portal.azu
...
 
 Windows
 ```sh
+cd .thunderbolt
```
Comment (timotheeguerin) [inline docs/00-getting-started.md]: I think the azb spark init should also rename that file for you and later(other pr) we can have it ask for the fields as they run `azb init`

Comment (jafreck) [reply docs/00-getting-started.md]: good point, it should definitely rename the file.

File: docs/00-getting-started.md [hunk 3]
```diff
@@ -53,15 +59,17 @@ Once you have one, simply log in and go to the [Azure Portal](https://portal.azu
...
 copy secrets.yaml.template secrets.yaml
 ```
 
 Linux and Mac OS
 ```sh
+cd .thunderbolt
 cp secrets.yaml.template secrets.yaml
```
Comment (jiata) [inline docs/00-getting-started.md]: cp? it should just give you secrets.yaml and not secrets.yaml.template

Comment (paselem) [reply docs/00-getting-started.md]: agreed. If we're going down the 'init' path then we should just create the secrets.yaml file. The template file will be distributed with our package (I suppose) and copied into the .thunderbolt on the init call.

Comment (jafreck) [reply docs/00-getting-started.md]: yeah, the latest commit fixed this already. Now `azb spark init` automatically renames the secrets file for you. These docs should be updated to reflect that.

File: dtde/spark/cli/init.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+
+def setup_parser(parser: argparse.ArgumentParser):
+    pass
+
+def execute(args: typing.NamedTuple):
+    config_src_path = constants.INIT_DIRECTORY_SOURCE
+    config_dest_path = constants.INIT_DIRECTORY_DEST
+
+    copy_tree(config_src_path, config_dest_path, update=1)
```
Comment (paselem) [inline dtde/spark/cli/init.py]: What happens if the source path already exists?

Comment (jafreck) [reply dtde/spark/cli/init.py]: Might be misunderstanding the question, but config_src_path exists when the repo is pulled. It is the config/ directory that is included in the repo.

config_dest_path is the .thunderbolt directory we are creating in the cwd. If that directory already exists and contains files that are in config_src_path, the files are left untouched. So running `azb spark init` twice in the same directory will not overwrite existing files.

**Pull request merged**